bw_projects/sqlite.py

```diff
@@ -1,14 +1,23 @@
-import pickle
+import json
 
-from peewee import BlobField, SqliteDatabase
+from peewee import TextField, SqliteDatabase
 
 
-class PickleField(BlobField):
+class JSONField(TextField):
```

Comment:
Please consider using the [native JSON field](https://docs.peewee-orm.com/en/latest/peewee/sqlite_ext.html#sqlite-json1); not sure if it makes more sense, but we should add a line about why we are rejecting it (if we are).

Reply:
Thank you @cmutel for this pointer, yes using the native field is better. Right now, I don't think we have specific requirement that is not covered by the native field.

I have now shifted the code to use this field.